libs/ui-lib/lib/ocm/components/clusterDetail/getProgressBarAlerts.tsx

```diff
@@ -47,6 +49,14 @@ export const getClusterProgressAlerts = (
         </StackItem>
       </Stack>
     );
+  } else if (finalizingStageTimeout) {
```

Comment:
What would happen if the installation takes a very long time, it first times out, and then the installation actually completes? 

1 - Would the API would still return  `finalizingStageTimedOut: true` ? Because we're adding this warning as an `else if`, none of the cluster details would be shown in that case.

2- Would the API have cleared `finalizingStageTimedOut` so that it's `false` ? Then there shouldn't be a problem.

I think it's worth double-checking with the Backend team to understand which of the two things would happen.

Reply:
> What would happen if the installation takes a very long time, it first times out, and then the installation actually completes?

It is a successful installation - like any other successful installation

> 1 - Would the API would still return `finalizingStageTimedOut: true` ? 

It should return `finalizingStageTimedOut: true` 
> 
> 2- Would the API have cleared `finalizingStageTimedOut` so that it's `false` ? Then there shouldn't be a problem.

Right